dags/challenge_dag_factory.py

```diff
@@ -50,7 +50,7 @@ def resolve_dag_config(challenge_name: str, dag_params: dict, config: dict) -> d
     
     # Start with default configuration
     dag_config = {
-        "schedule_interval": "*/3 * * * *",
+        "schedule_interval": "*/1 * * * *",
         "start_date": datetime(2024, 4, 9),
```

Comment:
Should this be 3 to prevent stuff like this from happening?

Reply:
My idea was to go back to tinkering with the resource management for the Airflow pods on Kubernetes with Bryan, but I can also revert this to 3 to be safe.

Reply:
I made [a ticket for this](https://sagebionetworks.jira.com/browse/DPE-1318). Not sure where to put it. Modify as you like!